parm/atm/utils/gdas_jediinc2fv3.yaml

```diff
@@ -0,0 +1,62 @@
+variable change:
+  variable change name: Model2GeoVaLs
+  input variables: &inputvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr,phis]
+  output variables: [ua,va,t,sphum,ice_wat,liq_wat,o3mr,delp,hydrostatic_delz]
+background:
+  geometry:
+    fms initialization:
+      namelist filename: ./fv3jedi/fmsmpp.nml
+      field table filename: ./fv3jedi/field_table
+    akbk: ./fv3jedi/akbk.nc4
+    layout:
+    - $(layout_x)
```

Comment:
let's go ahead and since this is a new YAML makes these all `{{ foo }}` since that is the direction we are moving towards